articles/setting-up-cicd-for-python-packages-using-github-actions/index.md

```diff
@@ -0,0 +1,279 @@
+# Setting up CI/CD for Python Packages using GitHub Actions
+
+## Outline
+
+- Prerequisites
+- What is CI/CD
+- Building a Python Package
+- Authenticating GitHub with Test PyPI
+- Packaging & Deploying with GitHub Actions
+- Testing Python Package Locally
+- Conclusion
+
+## Prerequisites
+
+- Understanding of git and GitHub
+- Creating a GitHub repository
+- Creating a Test PyPI account
+- Understanding how to [Build Python Packages](https://www.section.io/engineering-education/building-a-python-package-and-publishing-on-pypi/)
+
+## What is CI/CD
+
+CI/CD comprises three phrases: continuous integration, continuous delivery, and continuous deployment. These phrases seem similar but are of different meanings and implementations. Yet, they are very important to the software development life-cycle.
+
+### Continuous Integration
+
+Continuous Integration is a set of practices that enable development teams to integrate code into version control repositories regularly. According to Atlass*i*an, *It’s a primary DevOps best practice, allowing developers to frequently merge code changes into a central repository where builds and tests then run.*
```

Comment:
Suggestions:
- Why is `i` italicized in `Atlass*i*an`? If it's a typo please remove it.
- Instead italicizing the whole paragraph, you can alternatively, quote them like
> According to Atlassian, It’s a primary DevOps best practice, allowing developers to frequently merge code changes into a central repository where builds and tests then run.